src/main/java/seedu/duke/component/Resistor.java

```diff
@@ -0,0 +1,18 @@
+package seedu.duke.component;
+
+public class Resistor extends Component {
+    private static final String SI_UNIT = "μH";
+
+    public Resistor(double value){
+        super(value);
+    }
+    /**
+     * Returns the value and unit of Capacitor object.
+     *
+     * @return String of Capacitor value
```

Comment:
Consider changing this to Resistor object

Reply:
Thank you for pointing out!